src/task_1.2/task2.js

```diff
@@ -0,0 +1,28 @@
+const csv = require('csvtojson');
+const { pipeline } = require('stream/promises');
+const path = require('path');
+const fs = require('fs');
+const fsPromises = require('fs').promises;
+
+const readableStream = fs.createReadStream(path.join(__dirname, 'csv/nodejs-hw1-ex1.csv'));
+const writableStream = fs.createWriteStream(path.join(__dirname, 'txt/nodejs-hw1-ex1.txt'));
+
+const writeConvertedCsvToTxtFile = async () =>  {
+  if (!fs.existsSync(path.join(__dirname, 'txt'))) {
```

Comment:
Consider storing such expression results in a separate variable to improve readability.